src/types/FilteredBy.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilteredBy {
```

Comment:
Consider renaming the enum from 'FilteredBy' to a more descriptive name like 'TodoFilter'.